rpc/nns/hashes.go

```diff
@@ -27,3 +35,28 @@ func InferHash(sg ContractStateGetter) (util.Uint160, error) {
 
 	return c.Hash, nil
 }
+
+// ResolveFSContract is a convenience method that doesn't exist in the NNS
+// contract itself (it doesn't care which data is stored there). It assumes
+// that contracts follow the [ContractTLD] convention, gets simple contract
+// names (like "container" or "netmap") and extracts the hash for the
+// respective NNS record in any of the formats (of which historically there's
+// been a few).
+func (c *ContractReader) ResolveFSContract(name string) (util.Uint160, error) {
+	strs, err := c.Resolve(name+"."+ContractTLD, TXT)
+	if err != nil {
+		return util.Uint160{}, err
+	}
+	for i := range strs {
+		h, err := util.Uint160DecodeStringLE(strs[i])
+		if err == nil {
+			return h, nil
+		}
+
+		h, err = address.StringToUint160(strs[i])
+		if err == nil {
+			return h, nil
+		}
+	}
```

Comment:
Is it ever possible that the first record doesn't contain the desired hash in the LE format? I thought that it's always true for all contract: first record is LE string hash, second record is the same hash in the address format.

Reply:
It can be just one (any) of them and I'd like to move to NEP-proposed scheme in the future.